CMakeLists.txt

```diff
@@ -55,7 +55,7 @@ set (REVERSE_DOMAIN org.libelektra)
 
 set (KDB_VERSION_MAJOR 0)
 set (KDB_VERSION_MINOR 8)
-set (KDB_VERSION_MICRO 23)
+set (KDB_VERSION_MICRO 24)
```

Comment:
Because the jenkins build failed telling me that the KDB version was not increased. Obviously you gather more PRs until you really release a new version. I will revert that change.

Reply:
Weird, can you please point me to the Jenkins log?